nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw navbar link logo homepage 130

# Description
Added link to the navbar logo. The link takes the user back to the top of the page.

## Type of change


# Testing Instructions

- fetch then checkout into the branch. Run npm start to serve the website in a browser.
- Scroll down the web page to any area
- In the navbar, click the COHORT51 logo, and your screen should scroll to the top of the page.


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
